app/rest/controllers/src/main/java/io/syndesis/controllers/integration/online/ActivateHandler.java

```diff
@@ -104,7 +104,7 @@ public StateUpdate execute(IntegrationDeployment integrationDeploymentDefinition
             stepPerformer.perform("build", this::build, deploymentData);
             stepPerformer.perform("deploy", this::deploy, deploymentData);
         } catch (@SuppressWarnings("PMD.AvoidCatchingGenericException") Exception e) {
-            logError(integrationDeployment,"[ERROR] Activation failure");
+            logError(integrationDeployment, "[ERROR] Activation failure");
```

Comment:
Since we're catching `Exception` would it be better to pass the `Exception e` to `logError` to get the whole exception with stack trace logged?